src/e2e/java/teammates/e2e/pageobjects/AdminSessionsPage.java

```diff
@@ -0,0 +1,84 @@
+package teammates.e2e.pageobjects;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Page Object Model for the admin sessions page.
+ */
+public class AdminSessionsPage extends AppPage {
+
+    @FindBy(id = "btn-toggle-filter")
+    private WebElement toggleFilterButton;
+
+    @FindBy(id = "ongoing-sessions-table")
+    private WebElement ongoingSessionsTable;
+
+    public AdminSessionsPage(Browser browser) {
+        super(browser);
+    }
+
+    @Override
+    protected boolean containsExpectedPageContents() {
+        return getPageSource().contains("Ongoing Sessions");
+    }
+
+    public List<WebElement> getOngoingSessionsRows() {
```

Comment:
I think we should avoid returning WebElements as we want to minimize interaction outside of the page object.